packages/components/socialFeed/Map/Map.web.tsx

```diff
@@ -103,6 +110,43 @@ const MapManager = ({
   return null;
 };
 
+const DynamicMinZoom = ({
+  setMinZoom,
+}: {
+  setMinZoom: Dispatch<SetStateAction<number>>;
+}) => {
+  const map = useMap();
+
+  useEffect(() => {
+    const lat = 45;
+    const lng = 90;
+
+    const adjustedWorldMapBounds: LatLngBoundsLiteral = [
+      [-lat, -lng],
+      [lat, lng],
+    ];
```

Comment:
This part is independent of the Component  so we should put it outside of the component and we can use UPPER_CASE to define the constants

Reply:
we don't follow this convention, please use normal variable naming as in js, there is no real constants

we can put it outside of the func though

Reply:
fixed here: https://github.com/TERITORI/teritori-dapp/pull/1399/commits/b42b56d5fb01647b17aaa09df59351bb4f9f1532

Reply:
> we don't follow this convention, please use normal variable naming as in js, there is no real constants
> 
> we can put it outside of the func though

IMHO, if it's a constant and declared outside of function scope, it should be a UPPER_CASE.
if it's just a keyword const and declared in function scope, it's ok to define it with normal case.

Reply:
const in js don't exist
<img width="276" alt="Screenshot 2024-12-04 at 18 02 19" src="https://github.com/user-attachments/assets/ec6379ba-d628-4007-8c4d-255e1beab833">

Reply:
this makes refacto harder if the variable stops being a constant according to your definition
also we don't have a way to enforce this rule so this means more review work

Reply:
In the rust language it make more sense because constant are real constant and thus when you see UPPER_CASE, you have some guarantees. Even then it doesn't have much value IMO but at least it can be verified statically.

Reply:
we can vote on this